### ISSUE TITLE
nooelint: oelint.task.nocopy not working

### ISSUE THREAD

**Issue by DaniilKl** [priv-kweihmann/oelint-adv#691]
Used `oelint-adv` version: 6.7.1
Recipe:

```bb
(...)
do_install() {
    #install TA devkit
    install -d ${D}${includedir}/optee/export-user_ta/
    for f in ${B}/export-ta_${OPTEE_ARCH}/* ; do
        # nooelint: oelint.task.nocopy
        cp -aR $f ${D}${includedir}/optee/export-user_ta/
    done
}
(...)
```

`oelint-adv` results:

```bash
Advanced oelint..........................................................Failed
- hook id: oelint-adv
- exit code: 1

tempfile:3: SyntaxWarning: invalid escape sequence '\['
tempfile:3: SyntaxWarning: invalid escape sequence '\['
/home/danik/Projects/meta-layer/recipes-security/optee/optee-os-tadevkit_3.13.0.bb:12:error:oelint.task.nocopy:'cp' shall not be used in do_install. Use 'install' [branch:true]

```

**Comment by priv-kweihmann** [comment]
It has to be

```
# nooelint: oelint.task.nocopy
do_install() {
    #install TA devkit
    install -d ${D}${includedir}/optee/export-user_ta/
    for f in ${B}/export-ta_${OPTEE_ARCH}/* ; do
        cp -aR $f ${D}${includedir}/optee/export-user_ta/
    done
}
```

inline comments are not supported
